packages/material-ui/package.json

```diff
@@ -45,7 +45,6 @@
     "@types/react-transition-group": "^2.0.16",
     "clsx": "^1.0.2",
     "convert-css-length": "^2.0.0",
-    "debounce": "^1.1.0",
```

Comment:
This has two pros:

- 1 less dependency to handle
- a small win: (88,351 -> 88,270 gzipped)

Reply:
cc @TrySound https://github.com/component/debounce/pull/19.

Reply:
Nice, but I use this dependency in my project and this change will make my bundle slightly bigger.

Reply:
Let's ping debounce maintainers. Maybe my PR can be accepted.

Reply:
The module in question:
```jsx
export default function debounce(func, wait = 166) {
  let timeout;
  function debounced(...args) {
    const that = this;
    const later = () => {
      func.apply(that, args);
    };
    clearTimeout(timeout);
    timeout = setTimeout(later, wait);
  }

  debounced.clear = () => {
    clearTimeout(timeout);
  };

  return debounced;
}
```

Reply:
I have made a different commit 400469e as I wasn't sure it would make it to master.

Reply:
Ah, I see it has some nice defaults. Well, I'm ok if this works for you better. Not a big loss for me.

Reply:
I'm happy to revert, depending on people perception of the change. So, please +1 or -1. on it. I'm slightly leaning toward +1 so we can free ourselves from one dependency. I believe that most people use lodash.debounce anyway.